exporters/otlp/test/otlp_metrics_serialization_test.cc

```diff
@@ -5,6 +5,7 @@
 #include "opentelemetry/proto/metrics/v1/metrics.pb.h"
 
 #include <gtest/gtest.h>
+#include <vector>
```

Comment:
Why `std::list` works but not `std::vector`?

Reply:
Oh I don't remember, have remove the header now :)